java/arcs/android/storage/service/BindingContext.kt

```diff
@@ -125,6 +128,7 @@ class BindingContext(
                         }
                         resultCallback.onResult(null)
                     } catch (e: Throwable) {
+                        log.warning(e) { "Exception occurred while awaiting idle" }
```

Comment:
It's pretty hard to guarantee that the exceptions will never contain sensitive information unless we audit all places that throw exceptions. Maybe these log statements should be at `debug` level instead of `warning`?

Reply:
Aren't we going to need to continually do such an audit anyway? The number of causes where we catch-and-print is pretty small, in many cases we'll have uncaught exceptions that just print to the log.

My rationale here was that an exception at this point almost certainly indicates that we've actually done something wrong, so I wanted to elevate the importance of it. I might not be right about,  soI'll switch this to debug for now anyway, since it still gives us what we need.

Reply:
Firebase doesn't print debug-level messages :(

Reply:
@yuangu-google Is it possible for us to bump up the debug logging level on the firebase tests?